src/include/duckdb.h

```diff
@@ -2842,6 +2853,53 @@ If the append is successful, DuckDBSuccess is returned.
 */
 DUCKDB_API duckdb_state duckdb_append_data_chunk(duckdb_appender appender, duckdb_data_chunk chunk);
 
+//===--------------------------------------------------------------------===//
+// TableDescription
+//===--------------------------------------------------------------------===//
+
+/*!
+Creates a table description object.
+
+Note that the object must be destroyed with `duckdb_table_description_destroy`.
+
+* connection: The connection context.
+* schema: The schema of the table, or `nullptr` for the default schema.
+* table: The table name.
+* out: The resulting table description object.
+* returns: `DuckDBSuccess` on success or `DuckDBError` on failure.
+*/
+DUCKDB_API duckdb_state duckdb_table_description_create(duckdb_connection connection, const char *schema,
```

Comment:
The table description is completely separate from the default values - can we remove it from this PR as well?

Reply:
Moved `table_description` to <https://github.com/Tishj/duckdb/tree/c_table_description>